misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJob.kt

```diff
@@ -40,5 +46,6 @@ internal class SqsJob(
 
   companion object {
     const val ORIGINAL_TRACE_ID_ATTR = "x-original-trace-id"
+    const val IDEMPOTENCY_KEY_ATTR = "idempotencey_key"
```

Comment:
idempotency_key

Reply:
lol i probably triple checked the spelling